jest.teardown.ts

```diff
@@ -0,0 +1,25 @@
+import { log } from '@charmverse/core/log';
```

Comment:
i added this because my VS Code expected it to be here when it ran jest. maybe we could just use the teardown from scoutgame app but i didnt want to get bogged down in jest config